src/components/AssetsContainer.tsx

```diff
@@ -22,9 +21,7 @@ export const AssetsContainer: React.FC = () => {
 
   return (
     <div className={styles.container}>
-      <MarketPriceProvider>
-        {contracts ? <AssetTable contracts={contracts} /> : <div>Loading</div>}
-      </MarketPriceProvider>
+      {contracts ? <AssetTable contracts={contracts} /> : <div>Loading</div>}
```

Comment:
When the app loads the default value for price is `0`, then when we switch timeframe the value is `loading`. I think because is the same action happening in both cases it should have the same value. I'm thinking what should that be and I'm open for suggestions. 
Maybe just 'loading...' without the refresh icon for direction. Or just a loading gif.